phpBB/phpbb/storage/file_info.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\storage;
+
+use phpbb\storage\exception\not_implemented;
+
+class file_info
+{
+	/**
+	 * @var \phpbb\storage\adapter\adapter_interface
+	 */
+	protected $adapter;
+
+	/**
+	 * @var string
+	 */
+	protected $path;
+
+	/**
+	 * @var array
+	 */
+	protected $properties;
+
+	public function __construct($adapter, $path)
```

Comment:
Docblock pls

Reply:
This is provisional (how we get metadata from files), if you know a better way to do it tell me